packages/shared/src/utils/array/array.ts

```diff
@@ -1,9 +1,9 @@
-export const shuffleArray = array => {
```

Comment:
@yauheni-deriv, Just a question, Why did you decide to switch from the arrow function?

Reply:
I had error while trying to apply generic on arrow functions 
https://www.typescriptlang.org/docs/handbook/2/functions.html

Reply:
There shouldn’t be any issues 🤔 Although it doesn’t matter and we can move forward 🙇🏻 

```ts
export const shuffleArray = <T>(array: T[]): T[] => {
    for (let i = array.length - 1; i > 0; i--) {
        const j = Math.floor(Math.random() * (i + 1));
        [array[i], array[j]] = [array[j], array[i]];
    }

    return array;
};
```

Reply:
thanks. will change it. I guess I just missed something while refactoring. it works fine